NOTICE.txt

```diff
@@ -2,6 +2,23 @@ React-Spectrum
 =================
 The following is a list of sources from which code was used/modified in this codebase.
 
+-------------------------------------------------------------------------------
+This codebase contains a modified portion of code from Adobe which can be obtained at:
```

Comment:
is it really modified? we don't redistribute the code, we just wrap it right?

Reply:
hmm... that is correct, what do we say then? something like this?
`This codebase contains code from Adobe which can be obtained at`